test/library/packages/csv/CSV.chpl

```diff
@@ -69,7 +69,10 @@ module CSV {
     /* Read a CSV file with fields of the types given by
        the arguments to the function
      */
-    iter read(type t...) throws {
+    iter read(type t...) throws where t.size > 1 ||
```

Comment:
Would it make sense to wrap the where clause into a helper function? Maybe this list of cases to check will grow as we support more types?  I could be wrong.

Reply:
I improved the where clauses.